content/scripts/build.js

```diff
@@ -135,6 +139,19 @@ function extractLocale(source, folder) {
   return locale;
 }
 
+function getMetadata(source, folder) {
+  const metadataRaw = fs.readFileSync(path.join(folder, "index.yaml"));
+  const metadata = yaml.safeLoad(metadataRaw);
+  const wikiHistoryPath = path.join(folder, "wikihistory.json");
+  if (fs.existsSync(wikiHistoryPath)) {
+    const wikiMetadataRaw = fs.readFileSync(wikiHistoryPath);
+    const wikiMetadata = yaml.safeLoad(wikiMetadataRaw);
+    metadata.modified = wikiMetadata.modified;
+  }
```

Comment:
It's kinda messed up if this is the case (the `else` case) because we don't yet have a way to use `nodegit` to infer the `modified` from git history. 
That should probably be one of the next big tasks to work on. Granted, first we should probably do the all important `git add content/files` (but must remember to exclude all `index.hash` files.